examples/cmplog.rs

```diff
@@ -0,0 +1,31 @@
+fn main() {
+    // This fuzz harness demonstrates the capabilities of CmpLog.
+    // Simply run the fuzzer with the `-c 0` flag and it should find the crash immediately.
```

Comment:
I think the second line of this comment is no longer needed(?).

Reply:
Indeed, I modified it.